src/klicky.ts

```diff
@@ -1,3 +1,54 @@
 export const Klicky = () => {
   console.log("klickly ran!");
 };
+
+const outer = document.querySelector('.outer');
+
+const capture = {
+  capture : true
+};
+
+const noneCapture = {
+  capture : false 
+};
+
+const once = {
+  once : false 
+};
+
+outer?.addEventListener('click', onceHandler, once);
+
+function onceHandler(event)
+{
+  console.log("You have clicked the page");
+  write();
+  var date = new Date();
+  dateTime(date);
+}
+
+function write()
+{
+  const para = document.createElement("p");
+  para.innerText = "You have clicked the page!";
+  document.body.appendChild(para);
```

Comment:
Nice job with this bit! This is how I would write it

Reply:
Thanks!